exonum-java-binding/testkit/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.exonum.binding</groupId>
+    <artifactId>exonum-java-binding-parent</artifactId>
+    <version>0.6.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>exonum-testkit</artifactId>
+  <version>0.6.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>Exonum TestKit</name>
+  <url>https://exonum.com/doc/version/latest/get-started/test-service/</url>
+
+  <properties>
+    <ejb-core.nativeLibPath>${project.parent.basedir}/core/rust/target/debug
```

Comment:
I think since #818 it must be slightly different, @vitvakatu can advise

Reply:
It's okay, as we don't have tests with release mode at the moment